website/settings/defaults.py

```diff
@@ -1831,4 +1831,5 @@ def parent_dir(path):
     'bluebook-inline': 'bluebook'
 }
 
-PREPRINTS_ASSETS = '/static/img/preprints_assets/'
+OSF_CDN_DOMAIN = 'https://staging-cdn.osf.io'
```

Comment:
So I guess on Production it would be
```
OSF_CDN_DOMAIN = 'https://osf.io'
PREPRINTS_ASSETS = '/preprints/assets/osf-assets/files/preprints-assets/'
```